app/packs/src/components/sampleTaskInbox/SampleTaskInbox.js

```diff
@@ -77,62 +77,69 @@ const SampleTaskInbox = ({}) => {
     };
 
     return (
-      <Panel>
-        <Panel.Footer>
+      <Card>
+        <Card.Footer>
           <div style={style} ref={dropRef}>
             {text}
+            <br />
+            {subtext}
           </div>
-        </Panel.Footer>
-      </Panel>
+        </Card.Footer>
+      </Card>
     );
   };
 
-  let display_value = sampleTasksStore.inboxVisible ? 'block' : 'none';
+  const handleDrag = (e, ui) => {
+    const { x, y } = deltaPosition;
+    setDeltaPosition({
+      x: x + ui.deltaX,
+      y: y + ui.deltaY,
+    });
+  }
 
   return (
-    <Draggable
-      handle=".handle"
-      bounds="body"
-    >
-      <Panel
-        variant="primary"
-        className="sampleTaskInbox small-col col-md-6"
-        style={{
-          zIndex: 10, position: 'absolute', top: '70px', left: '10px', display: display_value,
-          maxHeight: '80%'
-        }}
-      >
-        <Panel.Heading className="handle">
-          <div className="row">
-            <div className="col-md-1">
+    <Draggable handle=".modal-header" onDrag={handleDrag}>
+      <div>
+        <Modal
+          show={sampleTasksStore.inboxVisible}
+          onHide={sampleTasksStore.hideSampleTaskInbox}
+          backdrop={false}
+          keyboard={false}
+          centered
+          size="xl"
+          className="draggable-modal-dialog"
+          dialogClassName="draggable-modal"
+          scrollable={true}
+          style={{
+            transform: `translate(${deltaPosition.x}px, ${deltaPosition.y}px)`,
+          }}
+        >
+          <Modal.Header closeButton>
+            <Modal.Title className="handle draggable-modal-stack-title">
               <SampleTaskReloadButton />
-            </div>
-            <div className="col-md-10">{openSampleTaskCount()} open SampleTasks</div>
-            <div className="col-md-1">
-              <Button
-                variant="danger"
-                size="sm"
-                onClick={sampleTasksStore.hideSampleTaskInbox}
-              >
-                <i className="fa fa-times" />
-              </Button>
-            </div>
-          </div>
-        </Panel.Heading>
-        <Panel.Body>
-          <div className="row sampleTaskCreationDropzones">
```

Comment:
I notice a lot of class names, that do not correspond to css rules.
They are especially weird because they are in camel-case.
I would remove them where ever possible.

Reply:
The classes are for code navigation. otherwise I have no proper way of knowing which Component renders which part of the generated html. (I had a horrible experience using the React DevTools, so this is my preferred way at the moment)

Reply:
Yeah I like finding my stuff in the DOM inspector too (it's way faster). Let's take that topic to our next team meeting. I think it's important, that we establish best practices that are not only useful for the original authors but also transparent for others.